docs/custom_loading.rst

```diff
@@ -121,6 +121,25 @@ can be utilized by referencing its name in the ``read`` method of the
 
     spec = Spectrum1D.read("path/to/data", format="my-format")
 
+.. _multiple_spectra:
+
+Loading Multiple Spectra
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+It is possible to create a loader that reads multiple spectra from the same
+file. For the general case where none of the spectra are assumed to be the same
+length, the loader should return a `~specutils.SpectrumList`. Consider the
+custom JWST data loader as an example:
+
+.. literalinclude:: ../specutils/io/default_loaders/jwst_reader.py
+    :language: python
+
+Note that by default, any loader that uses ``dtype=Spectrum1D`` will also
+automatically add a reader for `~specutils.SpectrumList`. This enables client
+code to call ``specutils.SpectrumList.read`` in all cases if it can't make
```

Comment:
(minor)
```suggestion
code to call ``specutils.SpectrumList.read()`` in all cases if it can't make
```
or alternatively, I think there's some magic rst syntax that would let you link to `NDIOMixin.read`?

Reply:
I'm not sure this is appropriate since `read` actually takes parameters, so it's not an accurate reflection of usage.

Reply:
That's true, *but* it's a common convention in sphinx docs that means "this is a fuction/method".  See for example https://docs.python.org/3/tutorial/modules.html#the-dir-function .  Normally this happens automatically when you do ``:func:`whatever` ``, and I'm just thinking you can "mock that up" here.  But if you get the magical linking working it's a moot poi nt